github/issues_comments.go

```diff
@@ -10,6 +10,9 @@ import (
 	"time"
 )
 
+// MaxCommentLen is the maximum length of a comment that github will accept without a validationg error.
+var MaxCommentLen = 65535
```

Comment:
In the x, it says:

x

Also, does this need to be exported? It looks to me that it's not helpful to be a part of API and can be made unexported. Any arguments for why it should be exported?